src/main/openapi/daam.yaml

```diff
@@ -982,7 +982,7 @@ components:
            items:
              type: object
              properties:
-               application-id:
+               resource:
```

Comment:
important: Can you please rename it to `datasetId`? I believe we call it like that on different endpoints.